TIME:     18:09:2023   16:02:19
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'TRUE WHERE longid == 71468443' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 15, in setFound
    cursor.execute("UPDATE ducks SET found TRUE WHERE longid == %s" % (id))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'TRUE WHERE longid == 71468443' at line 1



TIME:     18:09:2023   16:03:07
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'TRUE WHERE longid == 71468443' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 15, in setFound
    print("here")
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'TRUE WHERE longid == 71468443' at line 1



TIME:     18:09:2023   16:03:17
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'TRUE WHERE longid == 71468443' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 16, in setFound
    cursor.execute("UPDATE ducks SET found TRUE WHERE longid == %s" % (id))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'TRUE WHERE longid == 71468443' at line 1



TIME:     18:09:2023   16:04:48
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 16, in setFound
    cursor.execute("UPDATE ducks SET found = TRUE WHERE longid == %s" % (id))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1



TIME:     18:09:2023   16:07:30
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 16, in setFound
    cursor.execute("UPDATE ducks SET found = TRUE WHERE longid == %s;" % (id))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1



TIME:     18:09:2023   16:07:36
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 16, in setFound
    cursor.execute("UPDATE ducks SET found = TRUE WHERE longid == %s;" % (id))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1



TIME:     18:09:2023   16:07:46
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 16, in setFound
    cursor.execute("UPDATE ducks SET found = TRUE WHERE longid == %s;" % (id))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1



TIME:     18:09:2023   16:08:13
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 16, in setFound
    cursor.execute("UPDATE ducks SET found = TRUE WHERE longid == '%s';" % (id))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1



TIME:     18:09:2023   16:08:23
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 16, in setFound
    cursor.execute("UPDATE ducks SET found = TRUE WHERE longid == '%s';" % (id))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== 71468443' at line 1



TIME:     18:09:2023   16:08:29
TYPE:     Method error
REQUEST:     GET /setFound?id=71468443 HTTP/1.1
EXCEPTION:     1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== '71468443'' at line 1
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 16, in setFound
    cursor.execute("UPDATE ducks SET found = TRUE WHERE longid == '%s';" % (id))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\cursor.py", line 617, in execute
    self._handle_result(self._connection.cmd_query(stmt))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\opentelemetry\context_propagation.py", line 77, in wrapper
    return method(cnx, *args, **kwargs)
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 1069, in cmd_query
    result = self._handle_result(self._send_cmd(ServerCmd.QUERY, query))
  File "C:\Users\SebSm\AppData\Roaming\Python\Python310\site-packages\mysql\connector\connection.py", line 846, in _handle_result
    raise get_exception(packet)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '== '71468443'' at line 1



TIME:     25:09:2023   16:20:20
TYPE:     Method error
REQUEST:     GET /getDucks HTTP/1.1
EXCEPTION:     can only concatenate str (not "tuple") to str
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 23, in getDucks
    data += x
TypeError: can only concatenate str (not "tuple") to str



TIME:     25:09:2023   16:20:58
TYPE:     Method error
REQUEST:     GET /getDucks HTTP/1.1
EXCEPTION:     can only concatenate str (not "tuple") to str
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 23, in getDucks
    print(data)
TypeError: can only concatenate str (not "tuple") to str



TIME:     25:09:2023   16:21:08
TYPE:     Method error
REQUEST:     GET /getDucks HTTP/1.1
EXCEPTION:     can only concatenate str (not "tuple") to str
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 23, in getDucks
    print(data)
TypeError: can only concatenate str (not "tuple") to str



TIME:     25:09:2023   16:22:26
TYPE:     Method error
REQUEST:     GET /getDucks HTTP/1.1
EXCEPTION:     can only concatenate str (not "int") to str
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 24, in getDucks
    data += x[0]
TypeError: can only concatenate str (not "int") to str



TIME:     25:09:2023   16:23:23
TYPE:     Method error
REQUEST:     GET /getDucks HTTP/1.1
EXCEPTION:     can only concatenate str (not "int") to str
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 80, in handleRequest
    result = str(self.methods[methodName][0](**paramsDict))
  File "C:\Users\SebSm\projects\ProjectDUCK\server.py", line 24, in getDucks
    data += str(x[0])
TypeError: can only concatenate str (not "int") to str



TIME:     02:10:2023   12:04:55
TYPE:     Connection
REQUEST:     
EXCEPTION:     timed out
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 26, in start
    connSocket,sourceAddr = servSocket.accept()
  File "C:\Python310\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
TimeoutError: timed out



TIME:     02:10:2023   12:05:13
TYPE:     Connection
REQUEST:     
EXCEPTION:     timed out
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 26, in start
    connSocket,sourceAddr = servSocket.accept()
  File "C:\Python310\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
TimeoutError: timed out



TIME:     02:10:2023   12:26:04
TYPE:     Connection
REQUEST:     
EXCEPTION:     timed out
TRACEBACK:     Traceback (most recent call last):
  File "C:\Users\SebSm\projects\ProjectDUCK\webServer.py", line 26, in start
    connSocket,sourceAddr = servSocket.accept()
  File "C:\Python310\lib\socket.py", line 293, in accept
    fd, addr = self._accept()
TimeoutError: timed out



